tsdb/engine/tsm1/compact.go

```diff
@@ -205,12 +207,14 @@ func (c *DefaultPlanner) Plan(lastWrite time.Time) []string {
 		return nil
 	}
 
+	c.lastPlanCompactedFull = false
+
 	return tsmFiles
```

Comment:
To compact multiple sets concurrently, I was thinking this return value could be changed return a slice of `CompactionGroup` next.  That way `Plan` is still run serially, but each `CompactionGroup` could be run concurrently if needed.